File: location_service/LocationService.Core/Domain/Entity/ImageEntity.cs
namespace LocationService.Core.Domain.Entity
{
    public class ImageEntity
    {
        public long Id { get; set; }
        public long entityId { get; set; }
        public string Url { get; set; }
        public string EntityType { get; set; }
        public bool IsPrimary { get; set; }
    }
}

File: accommodation_service/src/main/java/huy/project/accommodation_service/core/domain/entity/ImageEntity.java
package huy.project.accommodation_service.core.domain.entity;

import lombok.*;

import java.util.Map;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class ImageEntity {
    private Long id;
    private String url;
    private Long entityId;
    private String entityType;
    private Boolean isPrimary;
    private Map<String, String> versions;
}
